manifests/update.pp

```diff
@@ -88,10 +88,19 @@
   validate_bool($reboot_required)
 
   # Windows update service.
-  service { $service:
-    ensure => 'running',
-    enable => true,
-  }
+  case $::kernelmajversion {
+    '6.3', '10.0' : {
```

Comment:
Will this line need updating/appending to when new Windows versions get released? If so, would it be possible to make this a conditional on versions after `>= '6.3'`?